Clinical-Genomics/scout
# Fix #1514: variant_type must be remembered.

This PR fixes a bug.

**How to test**:
1. on stage or local with research loaded (click request research in a demo instance, then run ```scout --demo load research```
1. with the current master, note that when using next page, first page or reset filters on a research page, variant_type resets to the default clinical.
1. with this PR, that is not supposed to happen: research stays research, clinical stays clinical. This should hold for both SNVs and SVs.

**Expected outcome**:
The functionality should be working

**Review:**
